examples/pwa-app/d2.config.js

```diff
@@ -0,0 +1,19 @@
+const config = {
```

Comment:
example app files start here - probably the most interesting one is `SectionWrapper.js`